it/src/test/java/com/expediagroup/streamplatform/streamregistry/it/StreamRegistryIT.java

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright (C) 2018-2019 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expediagroup.streamplatform.streamregistry.it;
+
+import java.io.IOException;
+import java.net.ServerSocket;
+
+import lombok.SneakyThrows;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.runner.RunWith;
+import org.junit.runners.Suite;
+import org.junit.runners.Suite.SuiteClasses;
+import org.springframework.boot.SpringApplication;
+import org.springframework.context.ConfigurableApplicationContext;
+import org.testcontainers.containers.FixedHostPortGenericContainer;
+
+import com.expediagroup.streamplatform.streamregistry.StreamRegistryApp;
+import com.expediagroup.streamplatform.streamregistry.it.helpers.Client;
+
+@RunWith(Suite.class)
+@SuiteClasses({
+    DomainTestStage.class,
+    SchemaTestStage.class,
+    StreamTestStage.class,
+    ProducerTestStage.class,
+    ConsumerTestStage.class,
+    ZoneTestStage.class,
+    InfrastructureTestStage.class,
+    StreamBindingTestStage.class,
+    ProducerBindingTestStage.class,
+    ConsumerBindingTestStage.class
+})
+public class StreamRegistryIT {
+  private static final int POSTGRES_PORT = randomPort();
+
+  public static Client client;
+  public static String url;
+
+  private static ConfigurableApplicationContext context;
+
+  @ClassRule
+  public static FixedHostPortGenericContainer postgreSQLContainer =
+      new FixedHostPortGenericContainer<>("postgres:latest")
+          .withEnv("POSTGRES_USER", "streamregistry")
+          .withEnv("POSTGRES_PASSWORD", "streamregistry")
+          .withEnv("POSTGRES_DB", "streamregistry")
+          .withFixedExposedPort(POSTGRES_PORT, 5432);
+
+  @BeforeClass
+  public static void before() {
+    String[] args = new String[]{
+        "--server.port=0",
+        "--schema.registry.url=http://schema-registry",
+        "--spring.datasource.url=jdbc:postgresql://localhost:" + POSTGRES_PORT + "/streamregistry",
```

Comment:
FYI it is recommended to not use "localhost", but `container.getContainerIpAddress()`.
See https://www.testcontainers.org/features/networking/#getting-the-container-ip-address

Also, there is no need for `randomPort()`, you can just use `withExposedPort(5432)` and `getMappedPort(5432)`


Last but not least, you can simply use [the JDBC support](https://www.testcontainers.org/modules/databases/#database-containers-launched-via-jdbc-url-scheme), remove the container definition and use the following JDBC URL:
```
jdbc:tl:postgresql:///streamregistry
```
(note that you may need to set `spring.datasource.driver-class-name=org.testcontainers.jdbc.ContainerDatabaseDriver` too)